pkg/minishift/util/iso_path.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	minishiftConstants "github.com/minishift/minishift/pkg/minishift/constants"
+	"net/url"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+func GetIsoPath(isoURL string) string {
```

Comment:
GetIsoTypeAndVersion!? Maybe even returning two strings?

Reply:
@hferentschik Not sure what advantage we will have if we return two strings separately as the function is not really putting any logic for extracting them.

Reply:
GetIsoCachingPath or something, but it does not have to return version as can be seen with any image as `unnamed`.